flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/BackPressureSampleableTask.java

```diff
@@ -22,13 +22,13 @@
 import org.apache.flink.runtime.executiongraph.ExecutionAttemptID;
 
 /**
- * Task interface used by {@link StackTraceSampleService} for back pressure tracking.
+ * Task interface used by {@link BackPressureSampleService} for back pressure tracking.
  */
-interface StackTraceSampleableTask {
+public interface BackPressureSampleableTask {
 
 	boolean isRunning();
```

Comment:
It seems a bit strange to have this method in the interface. This state should be covered in the specific implementation of `boolean isBackPressured()`. That means if the task is not running, it indicates non back pressure. So we do not need to explicitly judge the running state outside.